test/test_latex_to_pdf.rb

```diff
@@ -43,11 +43,15 @@ def test_broken_doc_halt_on_error
     end
   end
 
-  def test_broken_doc_overriding_halt_on_error
```

Comment:
Is writing incomplete content or "error" content to the pdf something you want to preserve?